facturacion_mexico/cfdi_recibidos/doctype/regla_item_cfdi_recibido/regla_item_cfdi_recibido.py

```
@@ -0,0 +1,15 @@
+import frappe
+from frappe import _
+from frappe.model.document import Document
+
+
+class ReglaItemCFDIRecibido(Document):
+	def validate(self):
+		self._validar_filtros()
+
+	def _validar_filtros(self):
```

Ruff (0.15.14)

[warning] Missing return type annotation for private function `_validar_filtros`

Add return type annotation: `None`

(ANN202)